Sources/CX10/xla_tensor_wrapper.cc

```diff
@@ -15,7 +15,7 @@
 #if defined(_WIN32)
 #define XLA_API __declspec(dllexport)
 #else
-#define XLA_API
+#define XLA_API __attribute__((visibility("default")))
```

Comment:
Could you please use the reserved spelling?  `__attribute__((__visibility__("default")))`

Reply:
Done